process/sync/shardblock.go

```diff
@@ -149,6 +149,26 @@ func (boot *ShardBootstrap) SyncBlock(ctx context.Context) error {
 	return err
 }
 
+func isErrGetNodeFromDB(err error) bool {
```

Comment:
perhaps we should rename this function: "isKnownClosingOrMissingError" ?

Reply:
If it is closingErr, this func will return false. So naming the func `isKnownClosing...`, but if it is closing we return false is misleading.